src/applications/post-911-gib-status/utils/helpers.jsx

```diff
@@ -179,9 +179,9 @@ export const backendErrorMessage = (
       <va-telephone tty contact="711" />
       ).
     </p>
-    <Link className="vads-c-action-link--green" to="/" hrefLang="en">
+    <a className="vads-c-action-link--green" to="/">
```

Comment:
I would recommend using the `va-link-action` component if possible since this Formation class will eventually be deprecated in favor of it: https://design.va.gov/storybook/?path=/docs/components-va-link-action--docs

Reply:
Ok sounds good. Merging this branch to fix the broken link in prod, then I'll make the recommended change